sandbox/main.lua

```diff
@@ -911,12 +915,16 @@ function setConstants()
 	addPlayerShip("Wesson",		"Chavez",		createPlayerShipWesson		,"J")
 	addPlayerShip("Wiggy",		"Gull",			createPlayerShipWiggy		,"J")
 	addPlayerShip("Yorik",		"Rook",			createPlayerShipYorik		,"J")
+	addPlayerShip("Szpieg",		"Ktlitan Breaker",	createPlayerShipSzpieg	,"W")
```

Comment:
Please put the player ships in alphabetical order by name (s comes before y) regardless of whether the player ships are intended to be controlled by a crew in real life or not (see "Skray")